### PR TITLE
339 Added information for IBM Style online

### PR DESCRIPTION
**Pull request by julian-cable**

### CODE REVIEW

File: en-US/Resources.xml [hunk 1]
```diff
@@ -52,6 +52,44 @@
 				<para>
 					<citetitle>IBM Style Guide</citetitle>. IBM Corporation, latest version.
 				</para>
+				<para>
+					Red Hat employees can now access the latest version of IBM Style online, at the following site:
```
Comment (daobrien) [inline en-US/Resources.xml:56]: Missing non-breaking space, and I don't think "IBM Style" is the correct title.
Do you need to include "at the following site" and then put the url in a separate para? How about just "at <url>"?

File: en-US/Resources.xml [hunk 2]
```diff
@@ -52,6 +52,44 @@
...
+					Red Hat employees can now access the latest version of IBM Style online, at the following site: 
+				</para>
+				<para>
+          			<ulink url="https://www.ibm.com/docs/en/ibm-style/">https://www.ibm.com/docs/en/ibm-style/</ulink>
+        		</para>
+				<para>
+          			Go to that web page, and click <guimenuitem>Log in to view content</guimenuitem>.
+        		</para>
+				<para>
+          			On the <guimenuitem>Log in to IBM</guimenuitem> page, in the <guimenuitem>IBMid</guimenuitem> field, enter your <guimenuitem>xxx@redhat.com</guimenuitem> email address and click <guimenuitem>Continue</guimenuitem>.
```
Comment (daobrien) [inline en-US/Resources.xml:65]: Need to review DocBook element use. This guide doesn't use the same reduced element set that GLS uses.

File: en-US/Resources.xml [hunk 3]
```diff
@@ -52,6 +52,44 @@
...
+        		</para>
+				<para>
+          			The <guimenuitem>IBM Style</guimenuitem> page shows the following categories:
```
Comment (daobrien) [inline en-US/Resources.xml:68]: Page names use `<guilabel>`. Refer to the DocBook Guide for correct usage of `<gui*>` elements.

File: en-US/Resources.xml [hunk 4]
```diff
@@ -60,7 +98,7 @@
 				</para>
 
 			</listitem>
-      <listitem>
+     		 <listitem>
 				<para>
 					<citetitle>Merriam-Webster Collegiate Dictionary</citetitle>
```
Comment (daobrien) [inline en-US/Resources.xml:103]: We actually need to update this to "Unabridged" and not "Collegiate".

**Review by daobrien — CHANGES_REQUESTED** [review]
A few nitpicks to fix up.

File: en-US/Resources.xml (hunk 1 above)
Comment (julian-cable) [reply en-US/Resources.xml:56]: The title in the PDF is "IBM Style Guide", while on the online site it is "IBM Style". 
I will continue to apply the PDF title for now; we might need to revisit in future if the PDF is discontinued.

**Review by daobrien — APPROVED** [review]

**Pull request merged**